src/Utilities/Requires.hpp

```diff
@@ -11,7 +11,7 @@
 namespace Requires_detail {
 template <bool B>
 struct requires_impl {
-  using template_error_types_failed_to_meet_requirements_on_template_parameters =
+  using template_error_types_failed_to_meet_requirements_on_template_parameters=
```

Comment:
When I apply clang-format this change gets reverted... maybe use `// clang-format off`, `// clang-format on` to ensure it persists?